src/plugins/hetero/executable_network.hpp

```diff
@@ -23,6 +23,15 @@
 #include "infer_request.hpp"
 #include "plugin.hpp"
 
+#include "src/plugin.hpp"
+#include "src/config.hpp"
+
+namespace ov {
+namespace hetero {
+class Plugin;
+}
+}  // namespace ov
```

Comment:
You don't need to have this forward declaration because you already include the header.

Reply:
Unfortunately, `ov::hetero::Plugin` can't be determined w/o forward declaration